src/hammer-vlsi/defaults.yml

```diff
@@ -158,6 +158,15 @@ vlsi.inputs:
   # - "power" - This obstruction type stops only power routing/straps inside it
   placement_constraints: []
 
+vlsi.submit:
+  # The submit command to use. "none", "local", or null will run on the current host. See hammer_submit_command.py for other options.
+  command: "none"
+  # This should be a List[Dict[str, Dict[str, Any]]]. The list substitutes settings in order of appearance, and the first Dict key is the command.
+  # The second dict key is the name of that command's setting, followed by whatever type it takes.
+  settings: [
+    { "none": {} }
```

Comment:
No way for the code to infer that empty dict = `local`/`lsf` etc have no settings?

Reply:
This was meant to be an example, but I don't have a strong attachment to it.

Reply:
Changed.